tests/orchestrations_test.go

```diff
@@ -826,6 +826,144 @@ func Test_RecreateCompletedOrchestration(t *testing.T) {
 	)
 }
 
+func Test_SingleActivity_ReuseInstanceIDIgnore(t *testing.T) {
```

Comment:
These are the same tests that we have in grpc_test.go, right?

Reply:
yes, those are the same tests we have in grpc_test.go, but we are not calling through Grpc in this file, so I think it would be better to add them here as well.